i18n/locales/zh-CN/hourofcode/promote/resources.md

```diff
@@ -160,6 +160,6 @@ nav: promote_nav
 
 诚挚地，
 
-[您的名字], [职位]
+[Your Name], [Title]
```

Comment:
I wonder why we'd lose the translation here?

Reply:
Looks like we didn't lose it, it just got changed to this in crowdin: https://crowdin.com/translate/hour-of-code/612/en-zhcn#109293

Reply:
Fun! Well not an error with sync, so ok to merge.